packages/playground/src/main.tsx

```diff
@@ -14,4 +14,4 @@ import "./style.css";
 };
 
 const root = createRoot(document.getElementById("root")!);
-root.render(<App />);
+root.render(<App/>);
```

Comment:
can you revert that change, formatter should format it the other way. I have a PR enforcing formatting for .tsx files which for some reason wasn't. Make sure you have the prettier extension installed in vscode.

Reply:
I ran rush format for prettier formatter, but it stayed the same. So, I went in and corrected it manually.